config

```diff
@@ -187,6 +187,14 @@ bindsym $mod+$i3-wm.binding.ws_next workspace next
 set_from_resource $i3-wm.binding.ws_next2 i3-wm.binding.ws_next2 Mod1+Right
 bindsym $mod+$i3-wm.binding.ws_next2 workspace next
 
+## Navigate // Next Workspace within Output // <><Ctrl> Tab ##
```

Comment:
Can we use `on` or '`in' here instead of `within`?

Reply:
'on' is I think the better choice of the two, so updated to on in all four places.